src/NuGet.LibraryModel/LibraryDependency.cs

```diff
@@ -11,8 +11,6 @@ public class LibraryDependency
 
         public LibraryDependencyType Type { get; set; } = LibraryDependencyType.Default;
 
-        public LibraryIdentity Library { get; set; }
```

Comment:
Nobody was getting or setting this :). If/when it is needed, we can easily bring it back.